json_rpc/src/filters/tests/main_filter_with_recovery_tests.rs

```diff
@@ -284,10 +286,7 @@ async fn should_handle_malformed_request_but_valid_json() {
     assert_eq!(rpc_response.id(), &Value::Null);
     assert_eq!(
         rpc_response.error().unwrap(),
-        &Error::new(
-            ReservedErrorCode::InvalidRequest,
-            "unknown field `a`, expected one of `jsonrpc`, `id`, `method`, `params` at line 1 column 4"
-        )
+        &Error::new(ReservedErrorCode::InvalidRequest, "Missing 'jsonrpc' field")
```

Comment:
This also changed. If I was very strict I would say we're changing the API since errors returned are now different.

Reply:
Yeah, but that's already the case compared to the 1.4.6 version.  The errors are now (hopefully) more useful, as well as being compliant to the standard.